<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicSoundEffectsWidget</class>
 <widget class="QWidget" name="MusicSoundEffectsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音效插件</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>360</width>
     <height>510</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>音效插件</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>360</width>
     <height>485</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <widget class="QComboBox" name="stateComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="eqLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>均衡器: </string>
    </property>
   </widget>
   <widget class="QLabel" name="volumeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>声音平衡: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="eqButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>427</y>
      <width>35</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>禁用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eqEffectButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>427</y>
      <width>35</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>调整</string>
    </property>
   </widget>
   <widget class="QLabel" name="volumeLeftlabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>左</string>
    </property>
   </widget>
   <widget class="QLabel" name="volumeRightLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>右</string>
    </property>
   </widget>
   <widget class="MusicClickedSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="MusicSoundEffectsItemWidget" name="BS2BWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>340</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="MusicSoundEffectsItemWidget" name="CrossfadeWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>135</y>
      <width>340</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="MusicSoundEffectsItemWidget" name="StereoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>340</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="MusicSoundEffectsItemWidget" name="LADSPAWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>285</y>
      <width>340</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="MusicSoundEffectsItemWidget" name="SOXWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>340</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicClickedSlider</class>
   <extends>QSlider</extends>
   <header>musicclickedslider.h</header>
  </customwidget>
  <customwidget>
   <class>MusicSoundEffectsItemWidget</class>
   <extends>QWidget</extends>
   <header>musicsoundeffectswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
